DXRModules/DeusEx/Classes/DXRAugmentations.uc

```diff
@@ -95,6 +95,8 @@ static function RemoveAug(DeusExPlayer player, Augmentation aug)
     }
 
     aug.Deactivate();
+    if (AugAqualung(aug) != None)
```

Comment:
Probably better to make this generic with something like:

bAlwaysActive && AugmentationLocation!=LOC_Default

rather than hardcoding Aqualung in.  Gives better future proofing (or other mod-proofing)

Reply:
Is it better to make it generic though? I felt it was safer to deal with this specific case than to guess about future ones.

Reply:
Yes, definitely.

Also, are you sure with this fix that Aqualung still works if you reinstall it again after removing it?  I think you might need a GotoState('Inactive') as well, so that it can properly activate again afterwards.

Reply:
As much as I like things being generic, I'm still wary here for the reason described. I want to avoid unintended consequences.

I tested installing and removing and installing and removing it. It seems to work as expected.

Reply:
Watch the state of the aug while installing/removing/reinstalling and you will see that it stays in the state "Active" after the first install the entire time.  While it may not have an effect in this case, it still doesn't seem correct to leave it in that state instead of sending it back to state Inactive.

As for genericness, there are only two augs in our code base that aren't Default and are bAlwaysActive (Aqualung and Power Recirc).  Power Recirc has existing checks that kind of acted as a safety net for this scenario, but still gets left in a kind of half-disabled state as well, so it would be good to correct for it as well.

The rest of your concerns can be covered off with testing (ie. you should only see behavioural changes with Aqualung and Power Recirc, and nothing else should hit your new code)